src/clients/python/experimental_api_v2/examples/simple_grpc_v2_class_client.py

```diff
@@ -176,7 +177,8 @@ def requestGenerator(input_name, output_name, c, h, w, format, dtype, FLAGS):
     inputs[0].set_data_from_numpy(batched_image_data)
 
     outputs = []
-    outputs.append(grpcclient.InferOutput(output_name, class_count=2))
+    outputs.append(grpcclient.InferOutput(output_name))
+    outputs[0].set_parameter("classification", FLAGS.classes)
```

Comment:
There has been some changes in the library API. You must set the 'class_count' in the InferOutput constructor. See  PR #1242

Reply:
Done